src/vs/editor/contrib/dropOrPasteInto/browser/copyPasteController.ts

```diff
@@ -685,11 +685,6 @@ export class CopyPasteController extends Disposable implements IEditorContributi
 			}
 		}
 
-		// Specific case for pasting code blocks.
```

Comment:
cc @justschen You can use `yieldTo` to suggest how an edit should be ordered. In this case we can yield to `text.plain` to push the attachment edit after normal text pasting